src/t_hash.c

```diff
@@ -2669,14 +2838,36 @@ int hfieldIsExpired(hfield field) {
 /*-----------------------------------------------------------------------------
  * Hash Field Expiration (HFE)
  *----------------------------------------------------------------------------*/
-/* Called during active expiration of hash-fields */
+static void propagateHashFieldDeletion(redisDb *db, sds key, char *field, size_t fieldLen) {
+    robj *argv[] = {
+        shared.hdel,
+        createStringObject((char*) key, sdslen(key)),
+        createStringObject(field, fieldLen)
+    };
```

Comment:
Note: Later, maybe we can add a check to avoid creating these objects if we are not going to propagate anything.